.github/workflows/depenencies.yaml

```diff
@@ -0,0 +1,54 @@
+name: Dependencies test
+
+on:
+  workflow_dispatch:
+    inputs:
+      dependencies_list:
+        description: 'Space-separated list of dependencies to update. Example: "era-compiler-common era-compiler-llvm-context"'
+        required: true
+        default: ''
+        type: string
+      dependencies_branch:
+        description: 'Branch of testing dependencies'
+        required: true
+        default: ''
+        type: string
+
+jobs:
+  dependencies-check:
+    runs-on: matterlabs-ci-runner
+    container:
+      image: ghcr.io/matter-labs/zksync-llvm-runner:latest
+
+    steps:
+      # Ref to checkout will be defined automatically from the event that triggered the workflow
+      - name: Checkout source
+        uses: actions/checkout@v4
+
+      - name: Patch dependencies
+        shell: bash -ex {0}
+        env:
+          CARGO_TOML: Cargo.toml
+          GH_ML_URL: https://github.com
+          # `www.github.com` is a workaround for Cargo issue with `patch` section from `git` source
+          # https://github.com/rust-lang/cargo/issues/5478
+          GH_ML_PATCHED_URL: https://www.github.com
+        run: |
+          for DEP in ${{ inputs.dependencies_list }}; do
+            CRATE=$(echo "${DEP}" | cut -d'/' -f2)
+            # Handle special cases
+            if [ ${DEP} == "matter-labs-forks/llvm-sys.rs" ]; then
```

Comment:
If we're not testing those, shall we remove?

Reply:
I don't think we should remove, because even if we will not be able to trigger workflows from the `matter-labs-forks`, we would like to have a capability to update `llvm-sys.rs` on-the-fly if it's required by the update.

For example, if we need to update it together with `era-compiler-common`, it will be nice to test it together.